apps/modernization-api/src/main/java/gov/cdc/nbs/patient/search/PatientSearchCriteriaSortResolver.java

```diff
@@ -29,6 +29,8 @@ private SortOptions asOption(final Sort.Order sorting) {
       case ADDRESS -> asSortOption(ADDRESS, "address.streetAddr1.keyword", order);
       case "birthTime" -> asSortOption("birth_time", order);
       case "city" -> asSortOption(ADDRESS, "address.city.keyword", order);
+      case "county" -> asSortOption(ADDRESS, "address.cntyText.keyword", order);
+      case "country" -> asSortOption(ADDRESS, "address.cntryText.keyword", order);
```

Comment:
suggestion:  State, County, and Country are not directly sortable by the UI, there is an address column that includes the Street address 1, Street address 2, city, state, and zip.  The `asOptions` method may have to be changed to return a `Stream<SortOptions>` so that multiple columns can be sorted for these types of fields